### PR TITLE
fix some bugs

### PR DESCRIPTION
**Pull request by kitagry**
SSIA

### CODE REVIEW

File: bqemulatormanager/emulator.py [hunk 1]
```diff
@@ -17,8 +17,7 @@ def __init__(self, project_name: str, port: int, launch_emulator: bool = True, d
             if is_port_in_use(port):
                 raise PortOccupiedError(f'port {port} is occupied.')
             self.running_flg = True
-            self.proc = subprocess.Popen(["bigquery-emulator", "--project", f"{project_name}", "--port", f"{port}"
-                                          "--log-level", f"{log_level}"], shell=True)
+            self.proc = subprocess.Popen(["bigquery-emulator", "--project", f"{project_name}", "--port", f"{port}", "--log-level", f"{log_level}"], shell=True)
```
Comment (yokomotod) [inline bqemulatormanager/emulator.py:20]: re-format only, right ?

**Review by yokomotod — APPROVED** [review]
thx!

File: bqemulatormanager/manager.py [hunk 2]
```diff
@@ -69,7 +69,7 @@ def load(self, data, path):
 
     def create_dataset(self, dataset_name: str, exists_ok=True):
         dataset = bigquery.Dataset(f'{self.project_name}.{dataset_name}')
-        self.client.create_dataset(dataset)
+        self.client.create_dataset(dataset, exists_ok=exists_ok)
```
Comment (hirosassa) [inline bqemulatormanager/manager.py:72]: This change looks OK, and I also think we should check this kind of bug automatically by using a linter.

Maybe this repo? https://github.com/nhoad/flake8-unused-arguments

**Review by hirosassa — COMMENTED** [review]
Looks good but I put a minor comment.

File: bqemulatormanager/emulator.py (hunk 1 above)
Comment (kitagry) [reply bqemulatormanager/emulator.py:20]: No, there was not `,` between `f"{port}"` and `"--log-level"`.

**Review by hirosassa — APPROVED** [review]

File: bqemulatormanager/manager.py (hunk 2 above)
Comment (kitagry) [reply bqemulatormanager/manager.py:72]: I added. 5c6b0ea

Comment (hirosassa) [reply bqemulatormanager/manager.py:72]: Thanks!

**Pull request merged**